### PR TITLE
Update readme image link

### PR DESCRIPTION
**Pull request by pdostal**

### CODE REVIEW

**Review by Martchus — APPROVED** [review]
Although phrasing it "Update …" would probably be more fair :-)

File: README.md [hunk 1]
```diff
@@ -159,4 +159,4 @@ When RabbitMQ is enabled, `openqa-mon` will connect to all configured hosts. If
 
 ![Screenshot of a terminal running openqa-revtui showing four failed jobs in purple and a couple of empty job groups](doc/openqa-revtui.png)
 
-You find a set of example configurations in the [review](_review) subfolder.
\ No newline at end of file
+You find a set of example configurations in the [review](_review) subfolder.
```
Comment (grisu48) [inline README.md:162]: Maybe I just don't see it, but what is the change in this line? :thinking:

**Comment by grisu48** [comment]
Also please sign the commit. I know that you can do that @pdostal :wink:

File: README.md (hunk 1 above)
Comment (pdostal) [reply README.md:162]: I think `\n` :smile:

**Comment by pdostal** [comment]
> Also please sign the commit. I know that you can do that @pdostal 😉

I was signed, just GitHub needed me to remove and readd my key as it has been edited (prolonged) in the mean time.

I also signed it off with `--signoff` just to be sure that it will meet all your expectations :smile:

**Review by grisu48 — APPROVED** [review]

**Pull request merged**